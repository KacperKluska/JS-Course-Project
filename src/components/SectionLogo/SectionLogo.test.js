import Sectionlogo from "./SectionLogo";

let wrapper;

test("SectionLogo snapshot", () => {
  wrapper = mount(<Sectionlogo>Test logo!</Sectionlogo>);
  expect(wrapper).toMatchSnapshot();
});
