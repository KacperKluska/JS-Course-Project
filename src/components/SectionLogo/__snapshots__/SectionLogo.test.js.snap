// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionLogo snapshot 1`] = `
<header
  className="sectionLogo"
>
  Test logo!
</header>
`;
